### PR TITLE
Simple splash menu button mouse hover...

### PR DESCRIPTION
**Pull request by mamarilmanson**
- Made Settings/Sort Dropdown/New button turn to pointing hand when hovering for UI feedback...
- Class Context and PopupWindow was returning error -> put them at 2nd line after "extends" after googling the problem...

### CODE REVIEW

**Comment by mamarilmanson** [comment]
Reverted context and popup_window after they fixed themselves locally...

**Comment by CyberGh0st-droid** [comment]
Hay mamarilmanson. do you have the correct Godot build/built binary for Vpuppr?. if you do I'd really like to know the process to building godot so that I could also work on this project. It'd be a great help if you could share the proper process to reproduce the Godot build I've asked in the issues tab but never got a response

**Comment by mamarilmanson** [comment]
1. Build Godot or Build Vpuppr? (I'm using Godot 4.1.1)
2. Which issue are you referring to?

File: screens/context.gd [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class_name Context
 extends Node
 
+
```
Comment (you-win) [inline screens/context.gd:4]: Next time, try to avoid committing empty changes like this

**Review by you-win — APPROVED** [review]

**Comment by you-win** [comment]
Thanks for the PR! Please manually squash your changes next time as requested in the [contributing doc](https://github.com/virtual-puppet-project/vpuppr/blob/master/CONTRIBUTING.md#pull-requests-pr)

**Comment by CyberGh0st-droid** [comment]
> 1. Build Godot or Build Vpuppr? (I'm using Godot 4.1.1)
> 
>     2. Which issue are you referring to?


so you're using godot 4.1.1 to make changes to Vpuppr?


I wan't to know how I can Make changes to Vpuppr with godot. in the repository it says to download the custom godot build. which I tried but it didn't work. I want to build vpuppr but I need the correct Godot build. and the one that's linked in the repository didn't work.

**Comment by CyberGh0st-droid** [comment]
Oh sorry I see it's been updated I only saw now. I'll try again later today and sorry to have bothered you. will let you know how it goes

**Pull request merged**